matchers/close_to.jai

```diff
@@ -0,0 +1,35 @@
+close_to :: (expected: $T, $tolerance: float = 0.001) -> *Close_To_Matcher(T) {
+   matcher := TNew(Close_To_Matcher(T));
+   matcher.tolerance = tolerance;
+   matcher.expected = expected;
+   return matcher;
+}
+
+matches :: (using m: *Close_To_Matcher($T), actual: T) -> bool {
+   return Math.abs(expected - actual) <= tolerance;
+}
+
+describe :: (using m: *Close_To_Matcher($T), description: *Description) {
+   info := type_info(T);
+   append(description, "a ");
+   append(description, info.type);
+   append(description, " close to ");
+   append_value(description, expected);
+}
+
+describe_mismatch :: (using m: *Close_To_Matcher($T), actual: T, description: *Description) {
+   append(description, "was ");
+   append_value(description, actual);
+   append_value(description, " (delta ");
```

Comment:
That should be `append` (since you don’t want to print a value but an explanation).